### PR TITLE
Slider Positioning Using Min/Max

### PR DESCRIPTION
**Pull request by rabberbock**
### 🤔 This is a ...

- [x] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

https://github.com/ant-design-blazor/ant-design-blazor/issues/1898

### 💡 Background and solution

When the Minimum of the Slider is not set to 0 (e.g. 15), the slider is empty up until the Minimum is reached. To fix/update this, the Slider needs to take the Minimum and Maximum into account when calculating the positioning of the slider's handles.

I show how this may be able to be implemented. Some open questions are, should this be configurable? What is the apprpriate behavior? 

This PR also fixes some initialization issues when passing in Parameters to the Slider. Calls `SetStyle` in `OnInitialized` so two way binding initializes correctly

This is still a work in progress, but wanted to get some initial thoughts. Thanks!

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by ElderJames** [comment]
@rabberbock  Thanks for contribute this.

File: site/AntDesign.Docs/Demos/Components/Slider/demo/Basic.razor [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<div>
     <Slider TValue="double" DefaultValue="35" Disabled="@_disabled"  />
-    <Slider TValue="(double, double)" DefaultValue="(20, 50)" Disabled="@_disabled" />
+    <Slider TValue="(double, double)" Value="_value" Min="15" Max="75" DefaultValue="_value" Disabled="@_disabled" />
```
Comment (anddrzejb) [inline site/AntDesign.Docs/Demos/Components/Slider/demo/Basic.razor:3]: I don't think you should replace this. Line. But you can add it. Maybe this:
```razor
<div>
    <Slider TValue="double" DefaultValue="35" Disabled="@_disabled"  />
    <Slider TValue="(double, double)" DefaultValue="(20, 50)" Disabled="@_disabled" />
    <Slider TValue="(double, double)" Value="_value" Min="15" Max="75" DefaultValue="_value" Disabled="@_disabled" Marks=@_marks />

    Disabled: <Switch Size="small" Checked="@_disabled" OnChange="(e)=>OnSwitch(e)" />
</div>

@code
{

    private bool _disabled;

    private (double, double) _value = (15, 75);

    private SliderMark[] _marks =
        {
            new SliderMark(15, "15"),
            new SliderMark(75, "75")
    };

    private void OnSwitch(bool args)
    {
        _disabled = args;
    }
}
```
This though will also point you to this issue:
image

Do you think you could fix it?

**Review by anddrzejb — CHANGES_REQUESTED** [review]

File: site/AntDesign.Docs/Demos/Components/Slider/demo/Basic.razor (hunk 1 above)
Comment (rabberbock) [reply site/AntDesign.Docs/Demos/Components/Slider/demo/Basic.razor:3]: @anddrzejb Yep, I will update the demo and look into the issue with the SliderMarks. Thanks!

Comment (rabberbock) [reply site/AntDesign.Docs/Demos/Components/Slider/demo/Basic.razor]: @anddrzejb I made those changes. Let me know if you have any other feedback. Thanks!

**Review by anddrzejb — CHANGES_REQUESTED** [review]
Now we have this:
image

Should be this:
image

When reversing, we have this (jumping - looks like values set 2 times)
jumpingSlider

Should behave like this:
correctSlider

**Comment by rabberbock** [comment]
@anddrzejb I updated the styling for the Slider when only a single value is being used and not a range. Let me know if you have any more feedback. Thanks!

I could not reproduce the jump though when reversing. I can take a look further if that is still happening.

**Comment by ElderJames** [comment]
BTW, I found there is a new feature in antd.

slider-draggable

**Review by anddrzejb — APPROVED** [review]
From my side looks good. 
Unless you feel up to the challenge to do the trackable range, I believe it is ready for merging (once you solve the conflicts).

**Comment by rabberbock** [comment]
@anddrzejb Thanks for reviewing! I will leave the draggableTrack for a separate PR if I get around to it.

**Review by ElderJames — APPROVED** [review]

**Review by ElderJames — APPROVED** [review]

**Comment by ElderJames** [comment]
Thank you @anddrzejb @rabberbock !

**Pull request merged**